mittab/apps/tab/outround_pairing_views.py

```diff
@@ -0,0 +1,639 @@
+            current_seed += 1
+
+        current_seed = 1
+        for nov_team in nov_teams:
+            if current_seed > nov_teams_to_break:
+                break
+
+            if BreakingTeam.objects.filter(team=nov_team[0]).exists():
+                continue
+
+            BreakingTeam.objects.create(team=nov_team[0],
+                                        seed=current_seed,
+                                        effective_seed=current_seed,
+                                        type_of_team=BreakingTeam.NOVICE)
+
+            current_seed += 1
+
+        outround_tab_logic.pair(BreakingTeam.VARSITY)
+        outround_tab_logic.pair(BreakingTeam.NOVICE)
+
+        return redirect_and_flash_success(
+            request, "Success!", path="/outround_pairing"
+        )
+
+    # See if we can pair the round
+    title = "Pairing Outrounds"
+    current_round_number = 0
+
+    previous_round_number = TabSettings.get("tot_rounds", 5)
+
+    check_status = []
+
+    msg = "All Rounds properly entered for Round %s" % (
+        previous_round_number)
+
+    ready_to_pair = "Yes"
+    ready_to_pair_alt = "Checks passed!"
+    try:
+        tab_logic.have_properly_entered_data(current_round_number)
+        check_status.append((msg, "Yes", "All rounds look good"))
+    except PrevRoundNotEnteredError as e:
+        ready_to_pair = "No"
+        ready_to_pair_alt = str(e)
+        check_status.append(
+            (msg, "No", "Not all rounds are entered. %s" % str(e)))
+    except ByeAssignmentError as e:
+        ready_to_pair = "No"
+        ready_to_pair_alt = str(e)
+        check_status.append(
+            (msg, "No", "You have a bye and results. %s" % str(e)))
+    except NoShowAssignmentError as e:
+        ready_to_pair = "No"
+        ready_to_pair_alt = str(e)
+        check_status.append(
+            (msg, "No", "You have a noshow and results. %s" % str(e)))
+
+    rooms = outround_tab_logic.have_enough_rooms_before_break()
+    msg = "N/2 Rooms available Round Out-rounds? Need {0}, have {1}".format(
+        rooms[1][1], rooms[1][0])
+    if rooms[0]:
+        check_status.append((msg, "Yes", "Rooms are checked in"))
+    else:
+        check_status.append((msg, "No", "Not enough rooms"))
+
+    return render(request, "pairing/pair_round.html", locals())
+
+
+def outround_pairing_view(request,
+                          type_of_round=BreakingTeam.VARSITY,
+                          num_teams=None):
+
+    if num_teams is None:
+        num_teams = TabSettings.get("var_teams_to_break", 8)
+
+        while not math.log(num_teams, 2) % 1 == 0:
+            num_teams += 1
+
+        return redirect("outround_pairing_view",
+                        type_of_round=BreakingTeam.VARSITY,
+                        num_teams=num_teams)
+
+    pairing_released = False
+
+    if type_of_round == BreakingTeam.VARSITY:
+        pairing_released = TabSettings.get("var_teams_visible", 256) <= num_teams
```

Comment:
I'm a bit confused by this tab setting. What's wrong with just a `pairing_released` tab setting?

Reply:
With these two tab settings there are a few benefits:

1) I can track whether varsity / novice are released separately (some tournaments will split them up if one is running late, etc)
2) The pairings view isn't for one round, but rather for all of outs (this is a nice 1 screen record for outrounds), thus if you just have a toggle, it's unclear which round you are showing and hiding